### ISSUE TITLE
About FLOPs

### ISSUE THREAD

**Issue by Xiangtaokong** [MzeroMiko/VMamba#7]
Thanks for your great work~

I am calculating the FLOPs and Parameters using the computation script (VMamba/analyze/get_flops.py) you provided.

This is the input ():
![image](https://github.com/MzeroMiko/VMamba/assets/34824976/f16830b8-46fd-4830-86d5-887a8a789e70)

It is the arch of Vmamba-Tiny(for classification), we should get Para: 22M and FLOPs:4.5G (from paper table1)

But I got:
![image](https://github.com/MzeroMiko/VMamba/assets/34824976/c0d11592-ef42-4b32-a133-24ff447c9043)
Is these any thing wrong? Thank you~

**Comment by Xiangtaokong** [comment]
Okay, I found this line, it is the key……
![image](https://github.com/MzeroMiko/VMamba/assets/34824976/51887aa9-c101-460a-bd40-a6c85b86f62f)
I recommend not to use this way of directly overwriting the name, after all, there is a VSSM above, LOL.

**Comment by MzeroMiko** [comment]
hhh yes. Thank you for your kind advise. We will watch out for that.

**Comment by Xiangtaokong** [comment]
> hhh yes. Thank you for your kind advise. We will watch out for that.

One more question

Do you count FLOPs as FLOPs or MACs？
I found there is a "/2" in /VMamba/classification/models/vmamba/vmamba.py 
![image](https://github.com/MzeroMiko/VMamba/assets/34824976/bdcc3ff9-0106-43c1-8f61-0a3604001751)
In this case, MACs rather than FLOPs will be calculated in this part. Is it consistent in the whole calculation process?

**Comment by MzeroMiko** [comment]
We counted flops following the rules in fvcore (actually the position you found "/2" is where we copied from fvcore, hhh)  ,  which is also the basic rule mmengine or mmcv follows. And through mmpretrain, we confirmed that is the rule that almost all the models follow.
We calculated the rest of the code all based on fvcore (which shows the same results as mmengine do). So we are sure it is consistent，not only with other operations in this model, but also with other famous published models.
